src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exceptions;
+
+/**
+ * Duke Exception.
+ */
+public class DukeException extends Exception {
```

Comment:
Since this is a parameter to a public function is this a large scope (can't use short forms)? here
(And for its subclasses)